.github/workflows/build-pyodide-package.yml

```diff
@@ -0,0 +1,108 @@
+name: Build pyodide arbor
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      PYODIDE_VERSION: 0.22.0a3
+      PYODIDE_RELEASE_URL: "https://github.com/pyodide/pyodide/releases/download"
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+        path: arbor_src
+    - name: get-xbuildenv
+      run: |
+        XBUILDENV_BASENAME=xbuildenv-${PYODIDE_VERSION}.tar.bz2
+        wget -nc "${PYODIDE_RELEASE_URL}/${PYODIDE_VERSION}/${XBUILDENV_BASENAME}"
+        tar -xvf ${XBUILDENV_BASENAME}
+        rm ${XBUILDENV_BASENAME}
+    - name: read-makefile
+      run: |
+        make -f xbuildenv/pyodide-root/Makefile.envs .output_vars | grep 'PY' | tee -a $GITHUB_ENV
+    - name: get-emsdk
+      run: |
+        wget https://github.com/emscripten-core/emsdk/archive/refs/heads/main.zip
+        unzip main.zip
+        rm main.zip
+    - name: setup-emsdk
+      run: |
+        ./emsdk-main/emsdk install ${{env.PYODIDE_EMSCRIPTEN_VERSION}}
+        ./emsdk-main/emsdk activate ${{env.PYODIDE_EMSCRIPTEN_VERSION}}
+    - name: build-modcc-host
+      run: |
+        mkdir -p arbor_src/build_modcc_host
+        cd arbor_src/build_modcc_host
+        cmake .. -DARB_USE_BUNDLED_LIBS=ON
+        make modcc -j 8
+    - name: build-libarbor-emscripten
+      run: |
+        ABS_TOOLCHAIN=$(realpath emsdk-main/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake)
+        ABS_MODCC=$(realpath arbor_src/build_modcc_host/bin/modcc)
+        sed -i.bak -E \
+            '/only been tested|nonexistent_file|unbalanced brace/d' \
+            arbor_src/ext/random123/include/Random123/features/gccfeatures.h
+        mkdir -p arbor_src/build_arborlib_em
+        cd arbor_src/build_arborlib_em
+        cmake .. \
+            --toolchain "${ABS_TOOLCHAIN}" \
+            -DCMAKE_INSTALL_PREFIX=$(realpath prefix) \
+            -DARB_USE_BUNDLED_LIBS=ON \
+            -DARB_MODCC="${ABS_MODCC}" \
+            -DARB_ARCH=generic
+        make -j 8
+        make install
+    - name: build-arbor-python-so-emscripten
```

Comment:
So this is a second build of libarbor? Why is that necessary?

Reply:
We're not building libarbor in the third step - we are linking the libarbor* files together with the pybind binding files to a single .so file that can be loaded by pyodide. Maybe could use some more explanation in the comments